script/GoerliDeployment.s.sol

```diff
@@ -23,7 +23,6 @@ abstract contract GoerliDeployment is Contracts, GoerliParams {
 
     // --- utils ---
     governor = 0x5bc1c8783b729eaCeEBfe01C220A12A8Dd704C15;
-    delegatee[OP] = governor;
```

Comment:
Just in case, `delegatee[OP]` is still set as `governor` in `GoerliParams._getEnvironmentParams()`.

Reply:
yes, i'm aware of it, that's why the line was rm :)